libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4447,9 +4461,11 @@ MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration(const mavlink_comm
 #endif
 
 #if AP_INERTIALSENSOR_ENABLED
+#if AP_AHRS_ENABLED
```

Comment:
is AHRS the right check to be adding here?

Reply:
We're just about to call a method on the singleton - this is the only protection I was trying to add here :-)

Reply:
This should already be OK? its calling the INS singleton?

Reply:
We are, but we've now removed that method if AHRS isn't enabled, so you get a link error here if you don't gate it.

The method's really mis-placed; the trim you're calibrating is the AHRS trim, *not* the INS trim....